doc/whats_new.rst

```diff
@@ -89,6 +89,8 @@ API
 
     - Deprecated and renamed :func:`mne.viz.plot_image_epochs` for :func:`mne.plot.plot_epochs_image` by `Teon Brooks`_
 
+    - `picks` argument has been added to :func:`mne.time_frequency.tfr_morlet`, :func:`mne.time_frequency.tfr_multitaper` by `Teon Brooks`_
```

Comment:
I'm pretty sure you need double-backticks on `picks` to get it to render properly here